apps/webClient/src/components/NavButton/index.tsx

```diff
@@ -43,6 +49,7 @@ const NavButton: React.FC<Props> = ({
   isLeft = false,
 }) => {
   const classes = useStyles();
+  let location = useLocation();
```

Comment:
Current javascript best practices dictate that you should only use `let` in rare cases.

The idea is that if you allow a variable to change, then it can potentially make debugging the value of a variable more difficult.  Its not a huge deal, but something I'd like to do and is very common in the industry at the time of writing.

Reply:
also, one nitpick, not necessary to change, but something I like to do:

`const { pathname } = useLocation();`

☝️ doing that can save keystrokes later in the function